README.md

```diff
@@ -133,6 +133,8 @@ You should see a JSON response like that below:
 }
 ```
 
+As you can see, the text summarizer not only gives the summary text but also filters out some special characters such as `-lrb- `, `-rrb-` and `@highlight`, etc, from the [Daily Mail](https://github.com/abisee/cnn-dailymail) dataset. 
```

Comment:
```suggestion
In this example, the text summarizer not only gives the summary text but also filters out some special characters such as `-lrb-` (representing `(`), `-rrb-` (representing `)`), and `@highlight`, etc, from the [Daily Mail](https://github.com/abisee/cnn-dailymail) dataset.
```

Reply:
I'd use `comparing to...` to show the difference.